Sources/BlogModuleApi/Objects/BlogCategory/BlogCategoryListObject.swift

```diff
@@ -10,21 +10,34 @@ import Foundation
 public struct BlogCategoryListObject: Codable {
 
     public var id: UUID
-    public var title: String
+    public var title: String?
```

Comment:
Why is the title & priority optional? They should always have values.

Reply:
For this, actually this is in the RestAPI return. The idea, is when it is deleted, you just return the timestamp, and ID, in ou case we don't return any more data, not even the title